2017/ECC/README.md

```diff
@@ -0,0 +1,25 @@
+## Conference: European Coreboot Conference
+
+* **Date**: 26-29.10.2017
+* **Location**: Bochum, Germany
+* **website**: -
+* **social media**: -
+
+### Presentation #1
+
+* **Topic**: Booting UEFI-aware OS on coreboot enabled platform - "in God's
+  Name, Why?"
+* **Presenters**: Piotr Król, Kamil Wcisło
+* **slides**: https://ecc2017.coreboot.org/uploads/talk/presentation/28/Booting_UEFI_aware_OS.pdf
```

Comment:
@artur-rs link is fine, but we should have PDF in this repo, because websites just disapear.